src/auth/oauth2/core/qgsauthoauth2method.cpp

```diff
@@ -70,6 +70,7 @@ QgsOAuth2Factory *QgsOAuth2Factory::instance()
   if ( !sInstance )
   {
     sInstance = new QgsOAuth2Factory();
+    qRegisterMetaType<QNetworkReply::NetworkError>( "QNetworkReply::NetworkError" );
```

Comment:
could we move this to QgsApplication::registerMetaTypes ? There's already some QNetworkReply / QNetworkRequest types in there.

Reply:
done